deployment-scanner/src/test/java/org/jboss/as/server/deployment/scanner/XmlCompletionScannerTest.java

```diff
@@ -40,16 +40,18 @@
 public class XmlCompletionScannerTest {
         
     private final FakeHandler handler;
+    private final org.jboss.logmanager.Logger lmLogger;
```

Comment:
I don't see any reason to use an `org.jboss.logmanager.Logger` here. Using a `java.util.logging.Logger` should work the same.

That said I'm somewhat surprised this test works at all. I would think a simple reference to the `DeploymentScannerLogger.ROOT_LOGGER`  should avoid the GC. If not then I'd say we need to change the test as the test container is using a different log context.